holoviews/util/transform.py

```diff
@@ -0,0 +1,293 @@
+import operator
+try:
+    from itertools import zip_longest as zip_longest
+except:
+    from itertools import izip_longest as zip_longest
+
+import numpy as np
+
+from ..core.dimension import Dimension
+from ..core.util import basestring, unique_iterator
+from ..element import Graph
+
+
+def norm_fn(values, min=None, max=None):
```

Comment:
Can't this just be `norm`, and the same for `bin`?  I don't see what the `fn` buys here, and when I see that I expect to see a family of normalization functions, not just one simple normalizer.  I don't think there is any namespace clash with the `norm` and `bin` methods on dim, as these are just bare functions in the module namespace.

Reply:
I guess, but these are not functions a user would ever access so I see very limited benefit. If you think I should rename them I'll change it to `_norm` and `_bin`.